iOS/WAStickersThirdParty/AllStickerPacksViewController.swift

```diff
@@ -8,12 +8,12 @@
 
 import UIKit
 
-class AllStickerPacksViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+class AllStickerPacksViewController: UIViewController {
 
     @IBOutlet private weak var stickerPacksTableView: UITableView!
 
     private var needsFetchStickerPacks = true
-    private var stickerPacks: [StickerPack] = []
+    private var stickerPacks: [WAStickerPack] = []
```

Comment:
Weird how you changed all `StickerPack` references to `WAStickerPack` and `Sticker` to `WASticker` but when looking at your fork, it is still `StickerPack` and `Sticker`.

Reply:
Also, is there a reason behind the renaming? I personally think `Sticker` and `StickerPack` are just fine. It is only a WhatsApp sticker app after all.